torsiondrive/tests/test_reproduce_examples.py

```diff
@@ -0,0 +1,161 @@
+"""
+Regression test for the torsiondrive package to reproduce the examples
+"""
+
+import pytest
+import os
+import sys
+import subprocess
+import shutil
+import filecmp
+import json
+import numpy as np
+from torsiondrive.dihedral_scanner import DihedralScanner
+
+@pytest.fixture(scope="module")
+def example_path(tmpdir_factory):
+    """ Pytest fixture funtion to download the examples, decompress and return the path to the example/ folder """
+    # tmpdir_factory is a pytest built-in fixture that has "session" scope
+    tmpdir = tmpdir_factory.mktemp('torsiondrive_test_tmp')
+    tmpdir.chdir()
+    example_version = '0.9.5'
+    url = f'https://github.com/lpwgroup/torsiondrive_examples/archive/v{example_version}.tar.gz'
+    subprocess.run(f'wget -nc -q {url}', shell=True, check=True)
```

Comment:
This is a clever way to pull down test data. I wonder if it would be safer to use Python's `urllib` or `urllib2` module here. I was surprised that my most recent mac laptop did not have `wget` installed by default.

Reply:
Thanks! I agree this could be improved later if we're facing issues with `wget`.